src/docfx/lib/schema/JsonSchema.cs

```diff
@@ -262,17 +262,29 @@ internal class JsonSchema
         /// </summary>
         public HashSet<string> DocsetUnique { get; set; } = new HashSet<string>();
 
+        /// <summary>
+        /// Whether content fallback is allowed for loc page
+        /// </summary>
+        public bool ContentFallback { get; set; } = true;
+
+        /// <summary>
+        /// Properties that are used to hold min item count that meet condition.
+        /// </summary>
+        [JsonConverter(typeof(ValueTupleConverter<JsonSchema, int>), "condition", "value")]
+        public (JsonSchema, int)[] MinItemsWhen { get; set; } = Array.Empty<(JsonSchema, int)>();
```

Comment:
Consider having a dedicated type and apply `OneOrManyConverter`?

Reply:
Ok, I've created a new type named 'ConditionalCheckSchema' for it.